libs/novui/src/json-schema-components/templates/ArrayFieldTemplate.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {
+  ArrayFieldTitleProps,
+  ArrayFieldTemplateProps,
+  getTemplate,
+  getUiOptions,
+  ArrayFieldTemplateItemType,
+} from '@rjsf/utils';
+import { Grid } from '@mantine/core';
+import { css } from '../../../styled-system/css';
+import { Box, Flex, styled } from '../../../styled-system/jsx';
+import { title } from '../../../styled-system/recipes';
+
+const Title = styled('h2', title);
+
+export function ArrayFieldTemplate(props: ArrayFieldTemplateProps) {
+  const { canAdd, disabled, idSchema, uiSchema, items, onAddClick, readonly, registry, required, title, schema } =
+    props;
+  const {
+    ButtonTemplates: { AddButton },
+  } = registry.templates;
+  const uiOptions = getUiOptions(uiSchema);
+  const ArrayFieldTitleTemplate = getTemplate('ArrayFieldTitleTemplate', registry, uiOptions);
+  const ArrayFieldItemTemplate = getTemplate('ArrayFieldItemTemplate', registry, uiOptions);
+
+  return (
+    <Box>
+      <ArrayFieldTitleTemplate
+        idSchema={idSchema}
+        title={uiOptions.title || title}
```

Comment:
```suggestion
        title={uiOptions.title ?? title}
```
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Should we allow `""` to work here from `uiOptions`?‏